src/components/Footer/Footer.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import './Footer.css';
+import InstaBW from '../../img/icons/InstaBW.png';
+import InstaC from '../../img/icons/InstaC.png';
+import MailBW from '../../img/icons/MailBW.png';
+import MailC from '../../img/icons/MailC.png';
+import TwitterBW from '../../img/icons/TwitterBW.png';
+import TwitterC from '../../img/icons/TwitterC.png';
+import HoverImage from 'react-hover-image';
+
+class Footer extends Component {
+  render() {
+    return (
+      <div>
+        <p>Monsrud Open &copy; {new Date().getFullYear()}</p>
+        <div className="box">
+          <div>
+            <a href="mailto:hello@monsrudopen.com" target="_blank">
+              <HoverImage src={MailBW} hoverSrc={MailC} style={{ width: 40 }} />
```

Comment:
Hadde vært nice om denne var delt opp over flere linjer slik som du har gjort med de to andre <HoverImage> komponentene